reana/reana_dev/git.py

```diff
@@ -371,8 +376,40 @@ def git_clone(user, component, exclude_components):  # noqa: D301
     if exclude_components:
         exclude_components = exclude_components.split(",")
     components = select_components(component, exclude_components)
+
+    instalation_directory = os.path.realpath(target_directory+'/')
+
+    def yes_no_question():
```

Comment:
If we choose the `-- target-directory` option way, then we don't really need to have the Y/N check. The CLI option should be sufficient in that case.

Reply:
Done